database/things/clox.pldb

```diff
@@ -1,7 +1,7 @@
 title clox
 type pl
-originCommunity https://github.com/evacchi
-country Italy 
 website https://craftinginterpreters.com/
```

Comment:
This is but one of many implementations of the Lox programming language described in the Crafting Interpreters website. Is it OK to have this one website associated with each possible implementation that might appear in the PLDB?

Reply:
IMO we should have one Lox entry, and then different entries for particular dialects.